core/commands/root.go

```diff
@@ -94,6 +94,7 @@ var rootSubcommands = map[string]*cmds.Command{
 	"mount":     MountCmd,
 	"name":      NameCmd,
 	"object":    ObjectCmd,
+	"patch":     PatchCmd,
```

Comment:
This patch plumbing should be `ipfs object patch`, so we can put some pretty sugar in `ipfs patch`.